spec.html

```diff
@@ -14438,6 +14444,11 @@ <h1>Runtime Semantics: Evaluation</h1>
           </table>
         </emu-table>
       </emu-clause>
+
+      <emu-note>
+        <p>Backing to the history the result of typeof *null* is "object" was mostly used for backward compatibility requirements. Meanwhile other additions allowed to express some parts of inheritance model throgh the value *null*. Today *null* used as a root of prototype chain for objects not being inherited from anything else. And since we maintain connection between *null* and *object*, we caution you against looking for some deep meaning in this behavior. Although it may in fact satisfy the reasons you need, careless combination of these capabilities might cause sunk cost.</p>
```

Comment:
As was [explained](https://github.com/tc39/ecma262/pull/1913#issuecomment-603895260) in #1913, the spec isn't the appropriate place for this kind of information. I'd suggest you look into MDN instead of making this change here.

(separately, i have no idea what you mean by "cause sunk cost")

Reply:
I need more data or information between.
If there is an ability to discuss it more...

Reply:
Good way is to try to figure out is to get rid of retrospection `typeof null === 'object` is a bad thing.
Because not it causes Catch-22
https://en.wikipedia.org/wiki/Catch-22

And this causes Sunk cost
https://en.wikipedia.org/wiki/Sunk_cost

And as I'm unable to explain this properly right now, it might be better to close both PRs
there should be the other way to explain, it might be spec as a source of truth, but obviously this should be done the other way, not by a note but by something else

Reply:
wrong link for Catch-22, the correct is
https://en.wikipedia.org/wiki/Catch-22_(logic)

So, yes, if it causes catch 22 or if it not causes -- it is the thing which is not applicable to the catch itself, because if there is a catch it doesn't matter, therefore we shoud get rid of any opinion about the sort of event caused
`typeof null === 'object'`

event itself is important, but giving opinion it was bad or good causes catch-22
and as I can see from Game theory - 
https://en.wikipedia.org/wiki/Game_theory

it is better to give good opinion if there is a way to give good
and it is better to insist on bad opinion if there is no other way

It is as simple as Nash equilibrium
https://en.wikipedia.org/wiki/Nash_equilibrium

That is it... but it is not technical, but it is how it works